atom/browser/atom_browser_client.cc

```diff
@@ -140,8 +185,7 @@ void AtomBrowserClient::OverrideSiteInstanceForNavigation(
     return;
   }
 
-  // Restart renderer process for all navigations except "javacript:" scheme.
-  if (url.SchemeIs(url::kJavaScriptScheme))
+  if (!ShouldCreateNewSiteInstance(browser_context, current_instance, url))
```

Comment:
@ahmedmohamedali new renderers created as a result of navigating from a sandboxed renderer should always be sandboxed.